keep/README.md

```diff
@@ -0,0 +1,74 @@
+# Agent Check: Keep
+
+## Overview
+
+[Keep][6] is an open source AIOps alert management and automation platform, which offers capabilities for consolidating, automating, and reducing noise across various parts of your observability stack. With this integration, you can harness the strength of both platforms, providing a unified and efficient approach to alert management and event correlation.
+
+Keep uses Datadog monitors data, logs, and events to correlate alerts and reduce noise.
```

Comment:
Hmm this sentence seems a bit off to me. Maybe:

"Keep uses Datadog monitor's data..."
"Keep uses Datadog's monitoring data..."

Reply:
Well, we specifically use the monitors data (monitors definition, events, etc.) so 🤷‍♂️ I can change that, let me know what makes sense.

Reply:
```suggestion
Keep uses Datadog's monitors, logs, and events data to correlate alerts and reduce noise.
```